index.html

```
@@ -158,13 +158,17 @@
 				<h3>EXAMPLES</h3>
 				<pre class="example">
 name.last        >> "Anderson"
+name.first       >> "Tom"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)